examples/electron/package.json

```diff
@@ -55,7 +55,8 @@
     "@theia/typescript": "^0.3.19",
     "@theia/userstorage": "^0.3.19",
     "@theia/variable-resolver": "^0.3.19",
-    "@theia/workspace": "^0.3.19"
+    "@theia/workspace": "^0.3.19",
+    "electron-store": "^2.0.0"
```

Comment:
I think if you put the dependency here then it won't work for other extensions.

Also, don't we need a CQ for the new dependency?

Reply:
Ok, moved that to core. 
I don't know about CQ. What is to do there?

Reply:
https://github.com/theia-ide/theia/wiki/Registering-CQs

Reply:
But let's ask @marcdumais-work whether we need a new CQ for this or not. Thanks!